cluster/src/main/java/org/apache/iotdb/cluster/config/ClusterDescriptor.java

```diff
@@ -89,112 +93,36 @@ public String getPropsUrl() {
     return url;
   }
 
-  public void replaceProps(String[] params) {
```

Comment:
so we will not support specify port by command line?

Reply:
yes